Resources/Locale/en-US/paper/paper-misc.ftl

```diff
@@ -51,7 +51,8 @@ book-text-agrichemkit-manual = Thank you for choosing the safe-for-all-ages Nano
       
       Each individual plant responds to unstable mutagen differently, so you may want to use small doses on multiple crops and try to crossbreed the best traits from each of those. Applying multiple doses to one plant can stack multiple changes and make it harder to single out desirable traits.
       Unstable mutagen is entirely safe when used as a fertilizer, and NanoTrasen takes no responsibility for dead crops, excessive water bills, newly sentient plants asking existential questions, or flora-strangled farmhands that may coincidentally occur while using it.
-      Do not drink unstable mutagen. Wash your hands thoroughly after handing. Wash your eyes if you have looked at unstable mutage for over 30 minutes in a 24 hour period. Store in a dark room between 293–295K. Do not use on corporate holidays. If you begin hearing voices telling you to drink unstable mutagen, please contact your doctor, head of personnel, or exorcist.
+      Do not drink unstable mutagen. Wash your hands thoroughly after handing. Wash your eyes if you have looked at unstable mutagen for over 30 minutes in a 24 hour period. Store in a dark room between 293–295K. Do not use on corporate holidays. If you begin hearing voices telling you to drink
```

Comment:
Did you mean to add this line break?

Reply:
Honestly it happened out of habit; I just saw the mis-matched line lengths.

Reply:
That line break should not be there mid-sentence since it will be formatted differently when printed in-game on a piece of paper.

Reply:
Ahh okay! Fixing once I get a fresh pull of the repo locally since I now see the issues of using the web editor (it added a newline in my last commit).

Reply:
Don't do web edits, kids :)